### DISCUSSION TITLE
Saving mesh_plot to image?

### DISCUSSION THREAD

**Discussion by 3deo-chris** [bwoodsend/vtkplotlib#9] [Q&A] [answered]
I'm trying to get an isometric view of an stl and save it to an image. Is there any way to orient the mesh to see all 3 planes? Any examples of how to use image_io.write to save a vpl.mesh_plot to a png or jpg?

**Comment by bwoodsend** [comment, marked as answer]
For setting camera angles use [view()](https://vtkplotlib.readthedocs.io/en/latest/Figures.html#vtkplotlib.view). There's nothing special dedicated to *isometric* views but it's only a two-liner anyway:
```python
vpl.view(camera_direction=[1, 1, 1])
vpl.reset_camera()
```

As for generating images, you save figures using [save_fig()](https://vtkplotlib.readthedocs.io/en/latest/Figures.html#save-fig).

Glueing those two together, your code should look something like:
```python
import vtkplotlib as vpl

# Plot the rabbit. Replace this with your own mesh.
vpl.mesh_plot(vpl.data.get_rabbit_stl())

# Set the camera angle. Toggle the minus signs to switch views.
vpl.view(camera_direction=[-1, 1, 1])
# Let VTK choose a sensible zoom.
vpl.reset_camera()

# Take an snapshot and save it.
vpl.save_fig("image.jpg", pixels=(600, 600), off_screen=True)

# Add as many view() and save_fig() calls as you need...

# Close the figure.
vpl.close()
```

I've used the rabbit for convenience although it's not a very good example because the model is at some odd angle.
